src/main/java/io/github/soc/directories/ProjectDirectories.java

```diff
@@ -42,68 +50,88 @@ public static ProjectDirectories fromPath(String path) {
     switch (operatingSystem) {
       case LIN:
       case BSD:
-        homeDir      = System.getenv("HOME");
-        cacheDir     = defaultIfNullOrEmptyExtended(System.getenv("XDG_CACHE_HOME"),  '/' + path_, homeDir + "/.cache/",       path_);
-        configDir    = defaultIfNullOrEmptyExtended(System.getenv("XDG_CONFIG_HOME"), '/' + path_, homeDir + "/.config/",      path_);
-        dataDir      = defaultIfNullOrEmptyExtended(System.getenv("XDG_DATA_HOME"),   '/' + path_, homeDir + "/.local/share/", path_);
+        homeDir      = System.getProperty("user.home");
+        cacheDir     = defaultIfNullOrEmptyExtended(System.getenv("XDG_CACHE_HOME"),  '/' + path, homeDir + "/.cache/",       path);
+        configDir    = defaultIfNullOrEmptyExtended(System.getenv("XDG_CONFIG_HOME"), '/' + path, homeDir + "/.config/",      path);
+        dataDir      = defaultIfNullOrEmptyExtended(System.getenv("XDG_DATA_HOME"),   '/' + path, homeDir + "/.local/share/", path);
         dataLocalDir = dataDir;
-        runtimeDir   = linuxRuntimeDir(homeDir, path_);
+        runtimeDir   = linuxRuntimeDir(homeDir, path);
         break;
       case MAC:
-        homeDir      = System.getenv("HOME");
-        cacheDir     = homeDir + "/Library/Caches/"              + path_;
-        configDir    = homeDir + "/Library/Preferences/"         + path_;
-        dataDir      = homeDir + "/Library/Application Support/" + path_;
+        homeDir      = System.getProperty("user.home");
+        cacheDir     = homeDir + "/Library/Caches/"              + path;
+        configDir    = homeDir + "/Library/Preferences/"         + path;
+        dataDir      = homeDir + "/Library/Application Support/" + path;
         dataLocalDir = dataDir;
         break;
       case WIN:
```

Comment:
Do the following changes cause any breakage for you, @alexarchambault and @wisechengyi?

Reply:
It should be ok, we're breaking binary compatiblity with coursier 1.0.x in the current master (1.1.x).

Reply:
Thanks! Looking at your last changes, you are only using `coursierDirs.cacheDir`, not `coursierDirs.configDir` or `coursierDirs.dataDir`, right?

If this is the case, I think no changes are required. What happened is that all paths of `ProjectDirectories` have sub-directories now (like cacheDir already had), increasing consistency, and hopefully reducing the potential of clashes for people with weird Windows setups.

Reply:
@alexarchambault Weird, it shows the changes above the lines I was asking about ... to be precise I'm interested in your opinion on lines 60 (old) to 73 (new). This should be the only thing that might be relevant to you.